search-app/src/cmr/search/services/content_service.clj

```diff
@@ -0,0 +1,153 @@
+(ns cmr.search.services.content-service
+  "Provides service functions for generating static content."
+  (:require
+   [clojure.string :as string]
+   [cmr.common-app.cache.consistent-cache :as consistent-cache]
+   [cmr.common-app.cache.cubby-cache :as cubby-cache]
+   [cmr.common.cache :as cache]
+   [cmr.common.cache.fallback-cache :as fallback-cache]
+   [cmr.common.cache.single-thread-lookup-cache :as stl-cache]
+   [cmr.common.config :refer [defconfig]]
+   [cmr.common.jobs :refer [defjob default-job-start-delay]]
+   [cmr.common.log :refer :all]
+   [cmr.common.util :as util]
+   [cmr.search.site.pages :as pages]
+   [cmr.search.site.static :as static]
+   [cmr.search.site.util :as site-utils]
+   [cmr.transmit.metadata-db :as mdb]))
+
+(def cache-key
+  "The key used to store the humanizer report cache in the system cache map."
+  :static-content-cache)
+
+(defconfig job-delay
+  "Number of seconds the `static-content-generator-job` needs to wait after
+  collection cache refresh job starts.
+
+  We need to add the delay so that the collection cache can be populated first
+  (collections are one of the key things queried when generating static
```

Comment:
Is this job actually using the collection cache? I don't think it actually needs the full collection metadata - just a few fields. I know the humanizer report is directly pulling the collections from the cache, but I think we only do that when we need to access the full metadata.

Reply:
Totally.

I've kept it, but set it to zero (and updated the docs), allowing us to control any delay we discover that we need in production.